### ISSUE TITLE
Is the `~/.aws/config` file required?

### ISSUE THREAD

**Issue by kenchan** [k1LoW/awsdo#30]
After setting up the AWS CLI, I encountered the following error when running awsdo. It appears that the ~/.aws/config file is not created immediately after setting up the AWS CLI.

```
$ AWS_PROFILE=myprofile awsdo -- aws s3api list-buckets
Error: open /home/MY_NAME/.aws/config: no such file or directory
```

I was able to execute it by creating an empty file with `touch ~/.aws/config`.

If it works with an empty file, maybe it shouldn’t give an error when the file isn’t there at all.

**Comment by k1LoW** [comment]
@kenchan Thank you for your report.

> If it works with an empty file, maybe it shouldn’t give an error when the file isn’t there at all.

You are right, this is missing from consideration.

I'll try to fix it.

**Comment by k1LoW** [comment]
Fixed version released as v0.12.0 ( Also, thank you for your sponsorship!!!!!!!! 🚀  )

**Comment by kenchan** [comment]
Thank you for your quick response! It's always helpful.
